app/schema/person/dog-owner.js

```diff
@@ -0,0 +1,50 @@
+const Joi = require('joi')
+
+const dogOwnerDogSchema = Joi.object({
+  breed: Joi.string(),
+  colour: Joi.string().allow('').allow(null),
+  dogReference: Joi.string(),
+  id: Joi.number(),
+  indexNumber: Joi.string(),
+  microchipNumber: Joi.string().allow('').allow(null),
+  microchipNumber2: Joi.string().allow('').allow(null),
+  name: Joi.string().allow('').allow(null),
+  sex: Joi.string().allow('').allow(null),
+  status: Joi.string(),
+  tattoo: Joi.string().allow('').allow(null),
+  birthDate: Joi.string().allow('').allow(null)
```

Comment:
birthDate will be a date

Reply:
yeah, was having issues thinking it was to do with this, seems it wasn't - changed back to date now